app.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+// list of variables used for
+var cookieCounter = 0;
+var cookieSales = 0;
+
+//math.floor(Math.random() * ((65 - 23) + 1) + 23 );
+
+var firstAndPike = {
+  name: 'First And Pike',
+  dailySales: [],
+  minimumHours: 23,
+  maximumHours: 65,
+  avgHours: 6.3,
+  randomGen: function () {
+    var random = Math.floor(Math.random() * (this.maximumHours - this.minimumHours + 1) + this.minimumHours);
+    return Math.round(random * this.avgHours);
+  }
+};
+for(var i = 0; i < 15 ; i++){
+  cookieSales = firstAndPike.randomGen();
+  firstAndPike.dailySales.push(cookieSales);
+}
+console.log(firstAndPike.dailySales);
+
+var seaTac = {
+  dailySales: [],
+  minimumHours: 3,
+  maximumHours: 24,
+  avgHours: 1.2,
+  randomGen: function () {
+    var random = Math.floor(Math.random() * (this.maximumHours - this.minimumHours + 1) + this.minimumHours);
+    return Math.round(random * this.avgHours);
+  }
+};
+//loop iteration begins for each hour from 6AM-8pm
+for(var i = 0; i < 15 ; i++){
+  cookieSales = seaTac.randomGen();
+  seaTac.dailySales.push(cookieSales);
+}
+
+console.log(seaTac.dailySales);
+
+// average hours *Randomly generated # will equal the sales for that hour which need to be looped 15 times from the items stored in an array.
+// generates a fake number of customers per hour
+//make a loop for 15 houts using the average hours and random gen. Store this data in an array
+
+var seattleCenter = {
+  dailySales: [],
+  minimumHours: 11,
+  maximumHours: 38,
+  avgHours: 3.7,
+  randomGen: function () {
+    var random = Math.floor(Math.random() * (this.maximumHours - this.minimumHours + 1) + this.minimumHours);
+    return Math.round(random * this.avgHours);
+  }
+};
+for(var i = 0; i < 15 ; i++){
```

Comment:
Where does the number 15 come from? Can you think of a clearer way to handle that?